content/buy-navcoin/index.zh.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "购买 NavCoin"
+date: 2018-03-20T08:09:08+13:00
+draft: false
+type: "default"
+author: ""
+description: "是公开开采的，没有代币销售。购买NavCoin的唯一方法是从第三方或交易所购买。"
+---
+{{< hero_section
+titleText="购买NavCoin"
+imgSrc="/images/buy-navcoin/buy-hero.svg"
+paragraphText="NavCoin是公开开采的，没有代币销售。获得资产净值的唯一方法是从第三方或交易所购买。交换提供了高度不同程度的安全性、安全性、隐私以及对您的资金和信息的控制。执行您自己的尽职调查，并选择一个钱包，您将保留您的NavCoin在选择一个交易所。请按此连结查阅兑换金额，以获取更多资讯。"
```

Comment:
This section sounds like it has been google translated or needs improving

Reply:
fixed